arch/aarch64/include/libucontext/bits.h

```diff
@@ -27,6 +27,7 @@ typedef struct libucontext_ucontext {
 	unsigned long uc_flags;
 	struct libucontext_ucontext *uc_link;
 	libucontext_stack_t uc_stack;
+	libucontext_sigset_t uc_sigmask;
```

Comment:
I don't think we actually declare `libucontext_sigset_t` anywhere?

Reply:
Musl defines `sigset_t` as:

```
typedef struct __sigset_t { unsigned long __bits[128/sizeof(long)]; } sigset_t;
```

That works out to 16 bytes on aarch64.

Reply:
I see, well it doesn’t seem to actually matter since it’s not used and is laid out after the regs.

Reply:
Hmm, but mcontext comes after the sigset.  __pad was intended to include the sigset here.

Reply:
This was an interesting bug, it turns out gcc was optimizing the struct layout in a way that matched my assumptions, but clang was not doing the same optimization.  So I fixed the padding size in b19027.